### ISSUE TITLE
Rollback on a specific migration, instead of stepping down to the target

### ISSUE THREAD

**Issue by jiMuBao** [cakephp/phinx#394]
Is is possible to only migrate down one of the earlier migration, rather than stepping down to it?

**Comment by quellUA** [comment]
I just changed datetime in the name of migration file and everything worked just fine

**Comment by RQuadling** [comment]
If your changes are independent then that would be OK, but if each subsequent migration is based upon changes in a previous migration, your sequence will be broken and rolling back further will be a problem.

**Comment by Sieg** [comment]
Still, the -force option or something like that for one specific migration could be very cool.

**Comment by lenton** [comment]
This would be very useful! I have required this ability a few times now.

I have just accidentally migrated a migration which had a few digits missing from its ID. Because of this, Phinx thought this was a really old migration and put it to the bottom of the stack.

With the ability to rollback a specific migration I could have easily rolled it back, fixed its ID, and then re-migrated. Instead I had to manually run the `down()` SQL and manually delete it from the `phinxlog` table.

I understand that chronological order is important for keeping integrity but I think some kind of command is needed for undoing mistakes.
